scripts/publish.js

```diff
@@ -122,11 +122,5 @@ shell.exec(`git add SHA256.md`);
 // Git commit
 shell.exec(`git commit -m "Updated SHA256"`);
 
-// Add formatting changes
```

Comment:
These steps now cause the script to stop/fail because there's nothing to add and commit as a result of the changes to the new prettier version. It used to be that we would loose some formatting when updating the package.json version number (https://github.com/forcedotcom/salesforcedx-vscode/commit/4445c6099bdcea45e5f405578bdb3ec72b63182a), that doesn't happen anymore (https://github.com/forcedotcom/salesforcedx-vscode/commit/3cd305db8d9de49d5966c73bfe958b47a3b32f29).